### PR TITLE
[AIRFLOW-1483] Making page size consistent in listing views

### PR DESCRIPTION
**Pull request by edgarRd**
Dear Airflow maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-1483] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-1483

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:

Views showing model listings had large page sizes which made page
loading really slow client-side, mostly due to DOM processing and
JS plugin rendering.
Also, the page size was inconsistent across some listings.

This commit introduces a configurable page size, and by default
it'll use a page_size = 100. Also, the same page size is applied to
all the model views controlled by flask_admin to be consistent.

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

This is a fairly simple change but verified manually that the configured page size is being used.

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

File: airflow/www/views.py [hunk 1]
```diff
@@ -90,6 +90,10 @@
 
 FILTER_BY_OWNER = False
 
+PAGE_SIZE = (conf.getint('webserver', 'page_size')
```
Comment (saguziel) [inline airflow/www/views.py]: if it is in default conf I don't think you need to do the `if` check

**Comment by saguziel** [comment]
Can you confirm that this changes the homepage paging size? I know I tried changing some stuff like this in the past and it didn't change the homepage

**Comment by edgarRd** [comment]
The home page is different, is not managed by `flask_admin`, it's a custom view implementation and  the paging is done client-side. In a following commit I'll match the functionality by doing it server side using this page size value for consistency.

**Comment by aoen** [comment]
LGTM

**Pull request closed without merging**